### DISCUSSION TITLE
Cover Art for touch screen in Beta 3.2.1

### DISCUSSION THREAD

**Discussion by bentonmischa** [MiczFlor/RPi-Jukebox-RFID#1962] [Q&A]
Hi there!

I am wondering where to put the cover.jpg image in Phoniebox Version 3 (3.2.1). In the old documentation I found out that the image has to be named cover.jpg and has to be placed inside the folder of the album. Unfortunately this does not seem to work.

Do I have to do it in a different way for Version 3?

Thanks a lot!

Mischa

**Comment by s-martin** [comment]
Coverart was broken, but is now part of the upcoming release 3.5.

you need to embed the covers into the mp3 files directly, no need to put a separate cover.jpg into the folder.
